recipes/django-localflavor/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "django-localflavor" %}
+{% set version = "3.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 24e9a03bc11ced92365821d3cfdd96eb4e2f3e44a6f40e86c645d3e0ec65ec53
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - localflavor
+
+about:
+  home: https://pypi.org/project/django-localflavor
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'django-localflavor is a collection of assorted pieces of code that are useful for particular countries or cultures. These are called the "local flavor" add-ons'
+
+  description: |
+  Most of the localflavor add-ons are localized form components deriving from the forms framework – for example, a USStateField that knows how to validate U.S. state abbreviations, and a FISocialSecurityNumber that knows how to validate Finnish social security numbers.
+  
+  doc_url: https://django-localflavor.readthedocs.io/
+  dev_url: https://github.com/django/django-localflavor/
+
+extra:
+  recipe-maintainers:
+    - django
```

Comment:
`django` is the name of the project and not a user who can have merge rights on this repository, so should be removed.

Reply:
done